cmd/geth/config.go

```diff
@@ -328,3 +338,49 @@ func setAccountManagerBackends(stack *node.Node) error {
 
 	return nil
 }
+
+func setDefaultMumbaiGethConfig(ctx *cli.Context, config *gethConfig) {
+	config.Node.P2P.ListenAddr = fmt.Sprintf(":%d", 30303)
+	config.Node.HTTPHost = "0.0.0.0"
+	config.Node.HTTPVirtualHosts = []string{"*"}
+	config.Node.HTTPCors = []string{"*"}
+	config.Node.HTTPPort = 8545
+	config.Node.IPCPath = utils.MakeDataDir(ctx) + "/bor.ipc"
+	config.Node.HTTPModules = []string{"eth", "net", "web3", "txpool", "bor"}
+	config.Eth.SyncMode = downloader.FullSync
+	config.Eth.NetworkId = 80001
+	config.Eth.Miner.GasCeil = 20000000
+	//--miner.gastarget is depreceated, No longed used
+	config.Eth.TxPool.NoLocals = true
+	config.Eth.TxPool.AccountSlots = 16
+	config.Eth.TxPool.GlobalSlots = 131072
+	config.Eth.TxPool.AccountQueue = 64
+	config.Eth.TxPool.GlobalQueue = 131072
+	config.Eth.TxPool.Lifetime = 90 * time.Minute
+	config.Node.P2P.MaxPeers = 200
+	config.Metrics.Enabled = true
+	// --pprof is enabled in 'internal/debug/flags.go'
+}
+
+func setDefaultBorMainnetGethConfig(ctx *cli.Context, config *gethConfig) {
+	config.Node.P2P.ListenAddr = fmt.Sprintf(":%d", 30303)
+	config.Node.HTTPHost = "0.0.0.0"
+	config.Node.HTTPVirtualHosts = []string{"*"}
+	config.Node.HTTPCors = []string{"*"}
+	config.Node.HTTPPort = 8545
+	config.Node.IPCPath = utils.MakeDataDir(ctx) + "/bor.ipc"
+	config.Node.HTTPModules = []string{"eth", "net", "web3", "txpool", "bor"}
+	config.Eth.SyncMode = downloader.FullSync
```

Comment:
```suggestion
	config.Eth.SyncMode = downloader.SnapSync
```

Reply:
Okay, So we will use SnapSync as default ?
I used full because that was the default in our start.sh